packages/flet/lib/src/controls/dismissible.dart

```diff
@@ -38,16 +38,19 @@ class _DismissibleControlState extends State<DismissibleControl> {
     bool disabled = widget.control.isDisabled || widget.parentDisabled;
     bool? adaptive =
         widget.control.attrBool("adaptive") ?? widget.parentAdaptive;
-    var contentCtrls = widget.children.where((c) => c.name == "content");
+    var contentCtrls =
+        widget.children.where((c) => c.name == "content" && c.isVisible);
 
     if (contentCtrls.isEmpty) {
-      return const ErrorControl("Dismissible does not have a content.");
+      return const ErrorControl(
+          "Dismissible.content must be provided and visible");
     }
 
-    var backgroundCtrls = widget.children.where((c) => c.name == "background");
+    var backgroundCtrls =
+        widget.children.where((c) => c.name == "background" && c.isVisible);
```

Comment:
Thought maybe we need another helper for `.children.where((c) => c.name == "<name>" && c.isVisible)`?

Reply:
I don't seem to get it.
Helper in which sense?